README.md

```diff
@@ -41,15 +41,15 @@ You can create up to 7 VFs on UHD Graphics 770
 4. Move the entire content of the repository to `/usr/src/i915-sriov-dkms-6.1`. The folder name will be the DKMS package name.
 5. Execute command `dkms -i -m i915-sriov-dkms -v 6.1`. `-m` argument denotes the package name, and it should be the same as the folder name which contains the package content. `-v` argument denotes the package version, which we have specified in the `dkms.conf` as `6.1`
 6. The kernel module should begin building.
-7. Once finished, we need to make a few changes to the kernel commandline. `nano /etc/default/grub` and change `GRUB_CMDLINE_LINUX_DEFAULT` to 'intel_iommu=on i915.enable_guc=7`, or add to it if you have other arguments there already.
+7. Once finished, we need to make a few changes to the kernel commandline. `nano /etc/default/grub` and change `GRUB_CMDLINE_LINUX_DEFAULT` to 'intel_iommu=on i915.enable_guc=3 i915.max_vfs=7`, or add to it if you have other arguments there already.
```

Comment:
This change should be more prominent. People who are upgrading from a previous release of `i915-sriov-dkms` may not notice this.

Reply:
OK, may add a note for updating from previous version on the top of README.

Reply:
You can also do this via by adding a udev rule to `/etc/udev/rules.d/intel-sriov.rules` if your OS uses udev.
```
ACTION=="add", DRIVER=="i915", ATTR{sriov_numvfs}="7"
```